src/main/java/com/github/cameltooling/lsp/internal/instancemodel/PathParamURIInstance.java

```diff
@@ -69,8 +70,11 @@ public String getValue() {
 	@Override
 	public CompletableFuture<List<CompletionItem>> getCompletions(CompletableFuture<CamelCatalog> camelCatalog, int positionInCamelUri, TextDocumentItem docItem, SettingsManager settingsManager) {
 		if(pathParamIndex == 0) {
-			if("kafka".equals(uriInstance.getComponentName())) {
+			String componentName = uriInstance.getComponentName();
+			if("kafka".equals(componentName)) {
```

Comment:
Wondering if "kafka" and "kamelet" should be constants somewhere?

Reply:
I was also wondering it :-)

not sure where to put them:
- in this class, as currently used only here (but they will surely be used elsewhere soon)
- in a new specific class ComponentNames
- Each in a specific kafka/Kamelet class. a new one or reusing a "CompletionProvider" one